stories/setup/webpack.config.babel.js

```diff
@@ -36,22 +38,11 @@ module.exports = {
       },
     ],
   },
-  // The following plugins are needed to help handle
-  // server-side imported components such as core/logger.
-  // Is this the best way??
-  plugins: [
-    new webpack.DefinePlugin({
-      CLIENT_CONFIG: JSON.stringify(getClientConfig(config)),
-    }),
-
-    new webpack.NormalModuleReplacementPlugin(
-      /config$/,
-      'core/client/config.js',
-    ),
-  ],
+  plugins: [...getPlugins({ includeCircularDependencyPlugin: false })],
   resolve: {
     alias: {
       normalize: 'normalize.css/normalize.css',
+      tests: path.resolve('./tests'),
```

Comment:
I'd like to see some duplication removed from this webpack config if possible, otherwise it will be a maintenance burden. We need to share as much common config as we can.

I made this change and everything seems to work. Could you apply this patch to share the common test alias?

 I tried using `getRules()` but webpack complains about schema errors which I don't understand. Duplicating the rules are OK for now but I'd like to see that fixed later on if possible.

```diff
diff --git a/stories/setup/webpack.config.babel.js b/stories/setup/webpack.config.babel.js
index d4f3f648c..4495c9847 100644
--- a/stories/setup/webpack.config.babel.js
+++ b/stories/setup/webpack.config.babel.js
@@ -4,9 +4,11 @@ import path from 'path';
 
 import autoprefixer from 'autoprefixer';
 
+import webpackConfig from '../../webpack.prod.config.babel';
 import { getPlugins } from '../../webpack-common';
 
 module.exports = {
+  ...webpackConfig,
   module: {
     rules: [
       {
@@ -39,11 +41,4 @@ module.exports = {
     ],
   },
   plugins: [...getPlugins({ includeCircularDependencyPlugin: false })],
-  resolve: {
-    alias: {
-      normalize: 'normalize.css/normalize.css',
-      tests: path.resolve('./tests'),
-    },
-    modules: [path.resolve('./src'), 'node_modules'],
-  },
 };

```

Reply:
I've cleaned this up as suggested. I had to use the prod config as you have here. Dev doesn't work b/c of certain components (like log, etc). 

Regarding the getRules...if we don't want to show prop types, then we can use this. I kind of find seeing the props types helpful so I looked into this a bit more and noticed if we did not include the babel-loader this works. So I basically have this set up as a config now for the function.

Reply:
Actually maybe I'll do a separate PR for this as part of this [issue](https://github.com/mozilla/addons-frontend/issues/6172)

Reply:
I did extract this out in a new PR: #6571